python/requirements/ml/tune-requirements.txt

```diff
@@ -11,3 +11,4 @@ hyperopt @ git+https://github.com/hyperopt/hyperopt.git@2504ee61419737e814e2dec2
 future
 nevergrad==0.4.3.post7
 optuna==3.2.0
+google-vizier[jax]
```

Comment:
It seems like this is causing some conflicts with the other pinned dependencies, which might require some upgrades. 

Are you able to access [this link](https://buildkite.com/ray-project/microcheck/builds/8400#0193b37a-a362-43d5-bd0a-a5d0b59eb5dd)?

Reply:
It seems the issue is our strict requirement on the `attrs` package version. I relaxed it in https://github.com/google/vizier/commit/1985bd97637a1dde5c83b73bdf91068e51372da6, hopefully should be resolved.